web/views.py

```
@@ -422,29 +440,56 @@ def enroll_course(request, course_slug):
+
+    # Create payment intent for enrollment
+    if discount_obj:
+        # Apply the discount and mark it as used
+        payment_amount = discounted_price
+        discount_obj.mark_as_used()
+        messages.success(
+            request, 
+            f"A ${discount_amount} discount has been applied to your enrollment thanks to your social media share!"
+        )
+    else:
+        payment_amount = course.price
```

Ruff (0.8.2)

Local variable `payment_amount` is assigned to but never used

Remove assignment to unused variable `payment_amount`

(F841)